tests/test_inverse.py

```diff
@@ -309,6 +314,8 @@ def test_inverse(self, _, data_name, acceptable_diff, *transforms):
 
         # Apply forwards
         for t in transforms:
+            if isinstance(t, Randomizable):
+                t.set_random_state(seed=get_seed())
```

Comment:
I think the issue is here, the randomised transforms are not deterministic unless the random state set.  this is a bit confusing because it's done in the `compose` by default... https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/compose.py#L100